website/docs/components/code-block/partials/code/how-to-use.md

```diff
@@ -0,0 +1,148 @@
+## How to use this component
+
+The basic invocation requires `@value` to be passed.
+
+```handlebars
+<Hds::CodeBlock
+  @value="aws ec2 --region us-west-1 accept-vpc-peering-connection"
+/>
+```
+
+### Title and description
+
+Optionally, you can pass a title and/or a description.
+
+```handlebars
```

Comment:
Probably, for sake of simplicity/brevity, I would do a single example with both title and description in the same code snippet

Reply:
I couldn't find relevant examples with both title and description but I replaced them with generic content in c80985a7361201cff38033cf3911c1d898e5e22b.